tools/project_manager/add_files.php

```diff
@@ -632,7 +633,8 @@ public function display()
         echo "<br>\n";
         echo _("Please review the information, and use the link at the bottom of the page to proceed with the load.");
         echo "<br>\n";
-        echo _("If there's a problem, return to the  <a href='$code_url/project.php?id=$projectid&detail_level=4'>Project Page</a> without loading the files.");
+        $project_url = project_page_link_url($projectid, ["detail_level=4"]);
+        echo _("If there's a problem, return to the  <a href='$project_url'>project page</a> without loading the files.");
```

Comment:
It was broken when you got here, but we should make this more translatale while we're here:
```php
echo sprintf(_("If there's a problem, return to the <a %s>project page</a> without loading the files."), "href='$project_url'");
```